### DISCUSSION TITLE
Multiple AppFlowyEditors in a ListView without a rigid size

### DISCUSSION THREAD

**Discussion by farosato** [AppFlowy-IO/appflowy-editor#348] [Q&A]
Hello 👋 
I’m trying to use the AppFlowyEditor in a Flutter app I’m building.
In a page of the app I have a ListView of editable notes. You can imagine something like a list of editable post it notes.
Currently the notes are simple TextFields.
What I’d like to do is replace the text fields with AppFlowyEditors. So each note in the list view would be an AppFlowyEditor and ideally on mobile I’d have just a single mobile toolbar on the page.

The part I'm having trouble with is the AppFlowyEditors sizing.
I’d like each editor to have a dynamic height matching the content it contains (just like a basic text field with no max lines set), but I’m having trouble doing this.
It seems that I have to specify a rigid height for the editors.
I tried setting shrinkWrap to true on the AppFlowyEditor instances and/or on the ListView but that still doesn't work.

I'm new to Flutter so I could be missing something simple but it seems that no matter the layout components and configurations I try I can't seem to make it work.

Could you please help or advise? 🙏
Thank you!

**Comment by LucasXu0** [comment]
Have you tried to wrap it with the `IntrinsicHeight` widget?

```dart
SingleChildScrollView(
          child: IntrinsicHeight(
            child: child,
          ),
        )
```

**Reply by microcoder-py** [reply]
Didn't work, SingleChildScrollView cannot render IntrinsicHeight widgets

**Comment by peter100u** [comment]
I plan to place multiple AppFlowyEditors in the listview, and the maximum height of the AppFlowyEditor is 5 rows. How to do this and how to set the 5 rows here

**Comment by peter100u** [comment]
""
Similar to Android development
 TextView
 Android: maxLines="5"
Android: ellip1size="end"
How to do the 5 lines here in the editor with ellipses

""
